### PR TITLE
feat(deleter): automatic tear down - EUBFR-83

### PR DESCRIPTION
**Pull request by kalinchernev**
Easier way to delete a staging/workspace for a given user. Useful in the end of the day or the end of the week.

Please run `yarn` to get new dependencies before running the tests.

2 new commands:
- `yarn run delete` to get rid of all `services`. As many ops are run in parallel to the AWS API, you might want to re-run the script twice if a longer operation, such as `elasticsearch` does not go through.
- `EUBFR_USERNAME=budg yarn run delete-demo` or similar to delete `server` apps for demos.

Notes:
- client apps are not delete-able in the same approach, because they are not CF stacks, and `serverless remove` does not have control over assets created by `serverless-finch`.
- dynamodb tables are preserved as usual

### CODE REVIEW

**Comment by raduchiriac** [comment]
I just ran the script and the deletion works fine for me

File: scripts/utils/deleteServerlessService.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  console.time(service);
+
+  const operation = spawn(
+    `./node_modules/.bin/sls`,
+    ['remove', `--stage ${config.stage}`, `--region ${config.region}`],
+    {
+      cwd: resolveSymbolicLink(`node_modules/@eubfr/${service}`),
+      // This is a helper for serverless.yml variables as an alternative to options
+      env: {
+        EUBFR_USERNAME: process.env.EUBFR_USERNAME
```
Comment (yhuard) [inline scripts/utils/deleteServerlessService.js]: `EUBFR_USERNAME: process.env.EUBFR_USERNAME || ''` is shorter

Comment (yhuard) [reply scripts/utils/deleteServerlessService.js]: As reported on slack, it only works in my environment if I remove this line. (`env: { ...`)

**Comment by kalinchernev** [comment]
@yhuard thanks for the updates, changes LGTM!

**Pull request merged**